tests/functional/adapter/test_grants.py

```diff
@@ -0,0 +1,6 @@
+import pytest
+
+from dbt.tests.adapter.grants.test_models import TestModelGrants
+
+class TestModelGrantsSnowflake(TestModelGrants):
+    pass
```

Comment:
A few formatting nits -- `pytest` was unused, there should be two spaces between imports and the following code, and newline at the end.

In case our automated processes didn't catch them, here they are:
```suggestion
from dbt.tests.adapter.grants.test_models import TestModelGrants


class TestModelGrantsSnowflake(TestModelGrants):
    pass

```

Reply:
Done